tools/rc/test/basic_test.bat

```diff
@@ -0,0 +1,39 @@
+:: This test is only invoked if the CMAKE flag US_USE_DETERMINISTIC_BUNDLE_BUILDS is set.
```

Comment:
nit: can the file be called something more descriptive? e.g., basic_deterministic_test.sh ?